app/grants/views.py

```diff
@@ -3173,3 +3174,249 @@ def contribute_to_grants_v1(request):
             'message': 'grant contributions recorded'
         }
     return JsonResponse(response)
+
+@login_required
+def ingest_contributions_view(request):
+    context = {
+        'title': 'Add missing contributions',
+        'EMAIL_ACCOUNT_VALIDATION': EMAIL_ACCOUNT_VALIDATION
+    }
+    if not request.user.is_authenticated:
+        return redirect('/login/github?next=' + request.get_full_path())
```

Comment:
this wouldn't be needed as `@login_required` is used

Reply:
Good point. Remove this check in def1e4a